src/screens/TopicPageScreen/index.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import _ from 'lodash';
 import SimpleToast from 'react-native-simple-toast';
-import {Animated, InteractionManager, Platform, StyleSheet} from 'react-native';
+import {Animated, Platform, StyleSheet} from 'react-native';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `InteractionManager` import has been removed. Ensure that this does not affect any other parts of the code where it might have been used.
```diff
- import {Animated, InteractionManager, Platform, StyleSheet} from 'react-native';
+ import {Animated, Platform, StyleSheet} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->